documentation-site/pages/components/card.mdx

```diff
@@ -28,6 +31,8 @@ content. Cards have a dynamic range of modifiers that allow text-only, text and
 and photography. All cards have an option to include text links or buttons as a primary action.
 The entire card can become an active click-target.
 
+<Yard componentName="Card" placeholderHeight={48} {...cardYardConfig} />
+
```

Comment:
Not sure if the Overrides and API sections are kept intentionally.

Reply:
it's still a work in progress PR, I'll remove that part